### DISCUSSION TITLE
Appreciation for Contributions

### DISCUSSION THREAD

**Discussion by Akshat111111** [Akshat111111/Hedging-of-Financial-Derivatives#1010] [General]
Thank you all @sanmarg @coderiksenthil @vivekvardhan2810 @aditya-bhaumik @sanskritiagr @zoey-11 @JagjeetChauhan @misskiran @pavitraag @priyankeshh @thevijayshankersharma @ashis2004 @Srijansarkar17 @aryan1165 @Shivani-Sharma-23 @alishasingh06 @Asymtode712 @PradnyaGaitonde @aryanchauh @shubha987 @AKSHITHA-CHILUKA @PRIYANSHU2026 @Pranshu-jais @Amarta113 
for your valuable contributions to this repository. Your efforts are greatly appreciated!

As we continue to grow and improve, I would like to request your support in voting for me as a project admin and mentor. Your vote would greatly help in advancing our project and fostering a collaborative environment.

**Comment by PRIYANSHU2026** [comment]
@Akshat111111  thank you sir for helping in our contribution journey

**Comment by Amarta113** [comment]
@Akshat111111 Had a great experience with open source community. Thank you all community members

**Comment by vivekvardhan2810** [comment]
@Akshat111111 It was a great experience and learnt many things regarding usage of finance through Machine learning sources . Thanking to all the opensource community members. And thank you sir for making the learning journey amazing.

**Comment by Akshat111111** [comment]
Dear Contributors-  @sanmarg @coderiksenthil @vivekvardhan2810 @aditya-bhaumik @sanskritiagr @zoey-11 @JagjeetChauhan @misskiran @pavitraag @priyankeshh @thevijayshankersharma @ashis2004 @Srijansarkar17 @aryan1165 @Shivani-Sharma-23 @alishasingh06 @Asymtode712 @PradnyaGaitonde @aryanchauh @shubha987 @AKSHITHA-CHILUKA @PRIYANSHU2026 @Pranshu-jais @Amarta113 
 Currently I am working on a project that can automate the entire Backend. You can also contribute in it as well-https://github.com/deskree-inc/blok .Kindly Star⭐ our repo as well as we are doing alot of hardwork for AI based integration and will soon release our paid open source mentorship program as well.
